src/test/java/com/exasol/releasedroid/main/ReleaseDroidTest.java

```diff
@@ -104,6 +104,15 @@ void testPlatformsFromConfig() {
         assertThrows(NullPointerException.class, () -> this.releaseDroid.run(userInput));
     }
 
+    @Test
+    void deprecatedPlatforms() {
+        final ReleaseConfig releaseConfig = ReleaseConfig.builder().releasePlatforms(List.of("jira")).build();
+        when(this.repositoryGatewayMock.getRepository(any())).thenReturn(this.repositoryMock);
+        when(this.repositoryMock.getReleaseConfig()).thenReturn(Optional.of(releaseConfig));
+        final UserInput userInput = builder().repositoryName("name").goal("RELEASE").build();
+        assertThrows(NullPointerException.class, () -> this.releaseDroid.run(userInput));
```

Comment:
Is it possible to throw something else than a NullPointerException? Maybe with a helpful message?